pkg/assembler/backends/ent/backend/neighbors_test.go

```diff
@@ -1,10 +1,40 @@
 package backend
 
-import "github.com/google/go-cmp/cmp"
+import (
+	"github.com/google/go-cmp/cmp"
+	"github.com/guacsec/guac/pkg/assembler/backends"
+)
 
 func (s *Suite) TestNode() {
-	s.T().Skip()
+	be, err := GetBackend(s.Client)
+	s.Require().NoError(err)
+
+	v, err := be.IngestArtifact(s.Ctx, a1)
+	check(s, be, v.ID, err, a1out)
+
+	p, err := be.IngestPackage(s.Ctx, *p1)
+	check(s, be, p.ID, err, p1out)
+	check(s, be, p.Namespaces[0].ID, err, p1out)
+	check(s, be, p.Namespaces[0].Names[0].ID, err, p1out)
+	check(s, be, p.Namespaces[0].Names[0].Versions[0].ID, err, p1out)
+
+	sc, err := be.IngestSource(s.Ctx, *s1)
+	check(s, be, sc.ID, err, s1out)
```

Comment:
This test isn't very easy to read when you abstract away the actual test logic. I'd prefer using table tests similar to the other types.

Reply:
I've implemented something consistent with other tests in `TestNodeNew` method for just 3 resources type.
If you could confirm this is the approach you were thinking of, then I'll add the tests for the other resources and I'll remove the first version of the `TestNode` method.